.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug Report
+description: File a bug report.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "**Required Info:**"
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.

```

Comment:
```suggestion
      description: Please search to see if an issue already exists for the bug you encountered. Please be mindful that some bugs may be attributable to individual ROS packages. 

```

Reply:
i just go ahead to borrow your suggestion.